install.sh

```diff
@@ -261,20 +261,17 @@ fi
 
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
```

Comment:
Compared to the original installer file for reference on the format and fixed a few other things, then compared to my previous commit. The file got reverted back to original for some reason so this should fix the formatting and do everything it was supposed to do previously. If there are any other features you want me to add or programs to install let me know. I read through the file one last time and nothing sprung out as being obsolete or needing to be removed anymore. Thing builds on a Docker Image fine using your repo.